content/pages/home.yaml

```diff
@@ -40,3 +40,18 @@ components:
   - component: projects
     title: Projects
     id: projects
+  - component: footer
+    title: Footer
+    brand: !g.string home.company_name.text
+    logo:
+      url: https://i.imgur.com/qrovqR0.png
```

Comment:
Let's reference our images from the /static folder.
We never want to rely on a third-party hosting with a few exceptions (Google CDN for 3rd party libraries so we can leverage caching for instance)

Reply:
Okay great, I was thinking that would be better but was having trouble figuring out where to place the images